sormas-backend/src/main/resources/sql/sormas_schema.sql

```diff
@@ -13249,4 +13249,12 @@ ALTER TABLE users_history ADD COLUMN externalid text;
 
 INSERT INTO schema_version (version_number, comment) VALUES (550, '#13080 Keycloak username modification is not synced by sormas');
 
+-- 2024-09-23 #13147 Phone Number Validation for E-Santé Reports – Remove and Store Non-Numeric Text
+ALTER TABLE externalmessage
+    ADD COLUMN personAdditionaldetails text;
```

Comment:
column names are all lowercase in general

Reply:
changed